Expand, correct, and tidy up XKB client manpages.  Correct typos in
xkbcomp's README.config file.

This patch by Branden Robinson.

Index: setxkbmap/setxkbmap.man
===================================================================
--- setxkbmap/setxkbmap.man.orig	2007-02-08 17:12:52.000000000 +0100
+++ setxkbmap/setxkbmap.man	2007-02-08 17:14:08.000000000 +0100
@@ -1,120 +1,167 @@
 .\"
 .\" $XFree86: xc/programs/setxkbmap/setxkbmap.man,v 3.6 2003/01/20 04:15:08 dawes Exp $
 .\"
-.TH SETXKBMAP 1 __xorgversion__
+.TH setxkbmap 1 __xorgversion__
 .SH NAME
-setxkbmap
-\- set the keyboard using the X Keyboard Extension
+setxkbmap \- set the keyboard using the X Keyboard Extension
 .SH SYNOPSIS
-.B setxkbmap [
-.I args
-.B ] [
-.I layout
-.B [
-.I variant
-.B [
-.I option  ...
-.B ] ] ]
+.B setxkbmap
+.RI [ option \ ...]
+.RI [ layout
+.RI [ variant
+.RI [ xkboption \ ...]\ ]\ ]
 .SH DESCRIPTION
 The
 .B setxkbmap
-command maps the keyboard to use the layout determined by the options
-specified on the command line.
-.P
+command configures the keyboard to use the layout determined by the
+parameters specified on the command line.
 An XKB keymap is constructed from a number of components which are compiled
-only as needed.  The source for all of the components can be found in
-.IR __projectroot__/lib/X11/xkb .
+only as needed.
+The source for all of the components can be found in
+.IR __XKBCONFIGROOT__ .
+.PP
+If one or more non\-option arguments are provided, the first is interpreted
+as a layout.
+.PP
+If two or more non\-option arguments are provided, the second is
+interpreted as the variant.
+.PP
+If three or more non\-option arguments are provided, the third and
+subsequent ones are interpreted as XKB options.
 .SH OPTIONS
-.TP 8
-.B \-help
-Prints a message describing the valid input to \fIsetxkbmap\fP.
-.TP 8
-.B \-compat \fIname\fP
-Specifies the name of the compatibility map component used to construct
-a keyboard layout.
-.TP 8
-.B \-config \fIfile\fP
-Specifies the name of an XKB configuration file which describes the
-keyboard to be used.
-.TP 8
-.B \-display \fIdisplay\fP
-Specifies the display to be updated with the new keyboard layout.
-.TP 8
-.B \-geometry \fIname\fP
-Specifies the name of the geometry component used to construct
-a keyboard layout.
-.TP 8
-.B \-keymap \fIname\fP
-Specifies the name of the keymap description used to construct
-a keyboard layout.
-.TP 8
-.B \-layout \fIname\fP
-Specifies the name of the layout used to determine the components which
-make up the keyboard description.  Only one layout may be specified on
-the command line.
-.TP 8
-.B \-model \fIname\fP
-Specifies the name of the keyboard model used to determine the components 
-which make up the keyboard description.  Only one model may be specified
-on the command line.
-.TP 8
-.B \-option \fIname\fP
-Specifies the name of an option to determine the components which make up 
-the keyboard description;  multiple options may be specified, one per
-\fI-option\fP flag. Note that
-.B setxkbmap
-summarize options specified in the command
-line with options was set before (saved in root window properties). If you
-want only specified options will be set use the \fI-option\fP flag with
-an empty argument first.
-.TP 8
+.TP
+.BR \-? ,\  \-help
+Print a usage message and exit.
+.TP
+.BI \-compat\  name
+Use the specified compatibility map component to construct a keyboard
+layout.
+.TP
+.BI \-config\  file
+Use the specified XKB configuration file, which describes the keyboard to
+be used.
+.TP
+.BI \-display\  display
+Update the specified display with the new keyboard layout.
+.TP
+.BI \-geometry\  name
+Use the specified geometry component to construct a keyboard layout.
+.TP
+.BI \-keymap\  name
+Use the specified keymap description to construct a keyboard layout.
+.TP
+.BI \-layout\  name
+Use the specified layout to determine the components which make up the
+keyboard description.
+Only one layout is permitted.
+.TP
+.BI \-model\  name
+Use the specified keyboard model to determine the components which make up
+the keyboard description.
+Only one model is permitted.
+.TP
+.BI \-option\  name
+Use the specified option to determine the components which make up the
+keyboard description; multiple options may be specified, one per
+.B \-option
+flag.
+Note that
+.B setxkbmap
+does not replace, but merely adds to, any XKB options already set on the
+display (these are saved in root window properties).
+If you want
+.I only
+the specified options to be set, use the
+.B \-option
+flag with an empty argument first.
+.TP
 .B \-print
-With this option the \fBsetxkbmap\fP just prints component names in a format
-acceptable by an \fBxkbcomp\fP (an XKB keymap compiler) and exits. The option
-can be used for tests instead of a verbose option and in case when one need
-to run both the \fBsetxkbmap\fP and the \fBxkbcomp\fP in chain (see below).
-.TP 8
-.B \-rules \fIfile\fP
-Specifies the name of the rules file used to resolve the request layout
-and model to a set of component names.
-.TP 8
-.B \-symbols \fIname\fP
-Specifies the name of the symbols component used to construct
-a keyboard layout.
-.TP 8
+Print component names in a format
+acceptable by
+.BR xkbcomp (1)
+(an XKB keymap compiler) and exit.
+This option is useful for testing, or when one needs to run both
+.B setxkbmap
+and
+.B xkbcomp
+in a pipeline (see below).
+.TP
+.BI \-rules\  file
+Use the specified rules file to resolve the requested layout and model to a
+set of component names.
+Only one rules file is permitted.
+.TP
+.BI \-symbols\  name
+Use the specified symbols component to construct a keyboard layout.
+.TP
 .B \-synch
-Force synchronization for X requests.
-.TP 8
-.B \-types \fIname\fP
-Specifies the name of the types component used to construct
-a keyboard layout.
-.TP 8
-.B \-variant \fIname\fP
-Specifies which variant of the keyboard layout should be used to determine 
-the components which make up the keyboard description.  Only one variant
-may be specified on the command line.
-.SH USING WITH xkbcomp
-If you have an Xserver and a client shell running on different computers and
-XKB configuration files sets on those machines are different you can get
-problems specifying a keyboard map by model, layout, options names.
-The thing is the \fBsetxkbcomp\fP converts these names to names of XKB
-configuration files according to files that are on the client side computer. 
-Then it sends the file names to the server where the \fBxkbcomp\fP has to
-compose a complete keyboard map using files which the server has.
-Thus if the sets of files differ significantly the names that the
-\fBsetxkbmap\fP generates can be unacceptable on the server side.  You can
-solve this problem running the \fBxkbcomp\fP on the client side too.
-With the \fI-print\fP option \fBsetxkbmap\fP just prints the files names
-in an appropriate format to its stdout and this output can be piped
-directly to the \fBxkbcomp\fP input. For example, a command
-
-\fBsetxkbmap us -print | xkbcomp - $DISPLAY\fP
-
-makes both step on the same (client) machine and loads a keyboard map into
-the server.
-.ig
-.SH SEE ALSO
-keyboards(1)
-..
+Force synchronization of X requests.
+.TP
+.BI \-types\  name
+Use the specified types component to construct a keyboard layout.
+.TP
+.BI \-variant\  name
+Use the specified layout variant to determine which components will make up
+the keyboard description.
+Only one variant is permitted.
+.TP
+.BI \-v\  level\fR,\ \fP \-verbose\  level
+Set verbosity to specified level.
+.SH "USING WITH xkbcomp"
+If an X server and X client are running on different computers and the XKB
+configuration files on those machines differ, problems may result when
+specifying a keyboard map.
+This is because
+.B setxkbmap
+converts the XKB component names to the names of XKB configuration files
+according to the XKB data on the client machine.
+It then sends the file names to the X server, where
+.B xkbcomp
+composes a complete keyboard map using
+.I its
+XKB configuration files.
+Thus, if the sets of files differ significantly, the names that
+.B setxkbmap
+generates on the client side may be unacceptable on the server side, or
+cause unexpected keyboard behavior.
+One way to solve this problem is by running
+.B xkbcomp
+on the client side as well.
+Use the
+.B \-print
+option of
+.B setxkbmap
+to print the XKB description
+in an appropriate format to standard output, and connect that to
+.BR xkbcomp 's
+standard input with a pipe.
+For example, the command
+.PP
+.nf
+.B setxkbmap \-print us | xkbcomp \- $DISPLAY
+.fi
+.PP
+generates both the XKB keyboard description
+.RB ( setxkbmap )
+and the compiled XKB keyboard map
+.RB ( xkbcomp )
+on the client side, and then uploads the map to the X server at
+.IR $DISPLAY .
 .SH FILES
-.I __projectroot__/lib/X11/xkb
+.I __XKBCONFIGROOT__
+.SH "SEE ALSO"
+.BR X (7),
+.BR xkbbell (1),
+.BR xkbcomp (1),
+.BR xkbevd (1),
+.BR xkbprint (1),
+.BR xkbvleds (1),
+.BR xkbwatch  (1)
+.SH COPYRIGHT
+Copyright 1994, Silicon Graphics Computer Systems and X Consortium, Inc.
+.br
+See
+.BR X (7)
+for a full statement of rights and permissions.
+.SH AUTHOR
+Erik Fortune, Silicon Graphics
Index: xkbcomp/README.config
===================================================================
--- xkbcomp/README.config.orig	2007-02-08 17:12:52.000000000 +0100
+++ xkbcomp/README.config	2007-02-08 17:14:08.000000000 +0100
@@ -100,7 +100,7 @@
 command loading the same keyboard mapping would look like:
 
      setxkbmap -rules xorg -model logicordless -layout "us,cz,de" \
-              -option "grp:alt_shift_toggle"
+               -option "grp:alt_shift_toggle"
 
 2.3  Even More Advanced Configuration
 
Index: xkbcomp/xkbcomp.man
===================================================================
--- xkbcomp/xkbcomp.man.orig	2007-02-08 17:12:52.000000000 +0100
+++ xkbcomp/xkbcomp.man	2007-02-08 17:14:08.000000000 +0100
@@ -5,105 +5,201 @@
 .\"
 .\" $XFree86: xc/programs/xkbcomp/xkbcomp.man,v 1.11 2003/07/28 21:57:02 herrb Exp $
 .\"
-.TH XKBCOMP 1 __xorgversion__
+.TH xkbcomp 1 __xorgversion__
 .SH NAME
 xkbcomp \- compile XKB keyboard description
 .SH SYNOPSIS
 .B xkbcomp
-[option] source [ destination ]
+.RI [ option \ ...]
+.I source
+.RI [ destination ]
 .SH DESCRIPTION
-.PP
 The
-.I xkbcomp
+.B xkbcomp
 keymap compiler converts a description of an XKB keymap into one of several
-output formats.   The most common use for xkbcomp is to create a compiled
-keymap file (.xkm extension) which can be read directly by XKB-capable X
-servers or utilities.   The keymap compiler can also produce C header
-files or XKB source files.  The C header files produced by xkbcomp can be
-included by X servers or utilities that need a built-in default keymap.
-The XKB source files produced by \fIxkbcomp\fP are fully resolved and can be 
-used to verify that the files which typically make up an XKB keymap
-are merged correctly or to create a single file which contains a complete
-description of the keymap.
+output formats.
+The most common use for
+.B xkbcomp
+is to create a compiled keymap file
+.RI ( .xkm
+extension) which can be read directly by XKB\-capable X servers or
+utilities.
+The keymap compiler can also produce C header files or XKB source files.
+The C header files produced by
+.B xkbcomp
+can be included by X servers or utilities that need a built\-in default
+keymap.
+The XKB source files produced by
+.B xkbcomp
+are fully resolved and can be used to verify that the files which typically
+make up an XKB keymap are merged correctly or to create a single file which
+contains a complete description of the keymap.
 .PP
-The \fIsource\fP may specify an X display, or an \fI.xkb\fP or \fI.xkm\fP
-file;  unless explicitly specified, the format of \fIdestination\fP
-depends on the format of the source.   Compiling a \fI.xkb\fP (keymap source) 
-file generates a \fI.xkm\fP (compiled keymap file) by default.   If the 
-source is a \fI.xkm\fP file or an X display, \fIxkbcomp\fP generates a 
-keymap source file by default.
+.I source
+may specify an X display, an
+.I .xkb
+file, or an
+.I .xkm
+file; unless explicitly specified, the format of
+.I destination
+depends on the format of the source.
+Compiling a
+.I .xkb
+(keymap source) file generates a
+.I .xkm
+(compiled keymap file) by default.
+If the source is a
+.I .xkm
+file or an X display,
+.B xkbcomp
+generates a keymap source file by default.
 .PP
-If the \fIdestination\fP is an X display, the keymap for the display
-is updated with the compiled keymap.
+If the
+.I destination
+is an X display, the keymap for the display is updated with the compiled
+keymap.
 .PP
-The name of the \fIdestination\fP is usually computed from the name
-of the source, with the extension replaced as appropriate.  When compiling
-a single map from a file which contains several maps, \fIxkbcomp\fP constructs
-the destination file name by appending an appropriate extension to the
-name of the map to be used.
+The name of the
+.I destination
+is usually computed from the name of the source, with the extension
+replaced as appropriate.
+When compiling a single map from a file which contains several maps,
+.B xkbcomp
+constructs the destination file name by appending an appropriate extension
+to the name of the map to be used.
 .SH OPTIONS
-.TP 8
+.TP
+.B \-?\fR, \fP-help
+Print a usage message and exit.
+.TP
 .B \-a
-Show all keyboard information, reporting implicit or derived information
-as a comment.  Only affects \fI.xkb\fP format output.
-.TP 8
+Show all keyboard information, reporting implicit or derived information as
+a comment; this option only affects
+.I .xkb
+format output.
+.TP
 .B \-C
-Produce a C header file as output (.h extension).
-.TP 8
+Produce a C header file as output
+.RI ( .h
+extension).
+.TP
+.B \-d
+Report debugging information (not supported in all builds).
+.TP
 .B \-dflts
 Compute defaults for any missing components, such as key names.
-.TP 8
-.B \-I\fIdir\fP
-Specifies top-level directories to be searched for files included by the 
-keymap description.  After all directories specified by \-I options
-have been searched, the current directory and finally, the default
-xkb directory (usually __projectroot__/lib/X11/xkb) will be searched.
+.TP
+.BI \-em1\  message
+Print
+.I message
+before displaying first error message (typically used by the X server).
+.TP
+.BI \-emp\  message
+Print
+.I message
+at the start of each error message line (typically used by the X server).
+.TP
+.BI \-eml\  message
+If there are any error messages, print
+.I message
+before exiting (typically used by the X server).
+.TP
+.BI \-I dir
+Search top\-level directory
+.I dir
+for files included by the keymap description.
+This option may be used multiple times.
+After all directories specified by
+.B \-I
+options have been searched, the current directory and finally, the default
+XKB data directory
+(usually
+.IR __projectroot__/lib/X11/xkb )
+will be searched.
 .sp
-To prevent the current and default directories from being searched,
-use the \-I option alone (i.e. without a directory), before any \-I
+To prevent the current and default directories from being searched, use the
+.B \-I
+option alone (i.e. without a directory), before any
+.B \-I
 options that specify the directories you do want searched.
-.TP 8
+.TP
 .B \-l
-List maps that specify the \fImap\fP pattern in any files listed on the
-command line (not implemented yet).
-.TP 8
-.B \-m\ \fIname\fP
-Specifies a map to be compiled from an file with multiple entries.
-.TP 8
+List maps that specify the
+.I map
+pattern in any files listed on the command line (not implemented yet).
+.TP
+.BI \-m\  name\fR,\ \fP \-map\  name
+Compile map from the file
+.IR name ,
+which is expected to contain multiple entries.
+.TP
 .B \-merge
-Merge the compiled information with the map from the server (not implemented
-yet).
-.TP 8
-.B \-o\ \fIname\fP
-Specifies a name for the generated output file.  The default is the name of 
-the source file with an appropriate extension for the output format.
-.TP 8
-.B \-opt\ \fIparts\fP
-Specifies a list of optional parts.  Compilation errors in any optional parts
-are not fatal.  Parts may consist of any combination of the letters \fIc\fP,
-\fIg\fP,\fIk\fP,\fIs\fP,\fIt\fP which specify the compatibility map, geometry,
-keycodes, symbols and types, respectively.
-.TP 8
-.B -R\fIdir\fP
+Merge the compiled information with the map from the server (not
+implemented yet).
+.TP
+.BI \-o\  name
+Direct output to the file
+.IR name .
+The default is the name of the source file with an appropriate extension
+for the output format.
+Specifying \(oq\-\(cq as
+.I name
+indicates standard output.
+.TP
+.BI \-opt\  parts\fR,\ \fP \-optional\  parts
+Compile keymap using a list of optional parts.
+Compilation errors in any optional parts are not fatal.
+.I parts
+may consist of any combination of the letters
+.BR c ,\  g ,\  k ,\  s ,\ and\  t
+which specify the compatibility map, geometry, keycodes, symbols and types,
+respectively.
+.TP
+.BI \-p\  count
+Strip
+.I count
+directories from filenames listed.
+.TP
+.BI \-R dir
 Specifies the root directory for relative path names.
-.TP 8
-.B -synch
-Force synchronization for X requests.
-.TP 8
-.B \-w\ \fIlvl\fP
-Controls the reporting of warnings during compilation.  A warning level 
-of 0 disables all warnings; a warning level of 10 enables them all.
-.TP 8
+.TP
+.B \-synch
+Force synchronization of X requests.
+.TP
+.BI \-v\  flags
+Set level of detail for listing.
+.I flags
+are as for the
+.B \-l
+option.
+.TP
+.BI \-w\  level
+Set the threshold of warnings during compilation to the specified
+.IR level .
+A level of 0 disables all warnings; a level of 10 enables them all.
+.TP
 .B \-xkb
-Generate a source description of the keyboard as output (.xkb extension).
-.TP 8
+Generate a source description of the keyboard as output
+.RI ( .xkb
+extension).
+.TP
 .B \-xkm
-Generate a compiled keymap file as output (.xkm extension).
+Generate a compiled keymap file as output
+.RI ( .xkm
+extension).
 .SH "SEE ALSO"
-X(__miscmansuffix__)
+.BR X (7),
+.BR setxkbmap (1),
+.BR xkbbell (1),
+.BR xkbevd (1),
+.BR xkbprint (1),
+.BR xkbvleds (1),
+.BR xkbwatch (1)
 .SH COPYRIGHT
 Copyright 1994, Silicon Graphics Computer Systems and X Consortium, Inc.
 .br
-See \fIX(__miscmansuffix__)\fP for a full statement of rights and permissions.
+See
+.BR X (7)
+for a full statement of rights and permissions.
 .SH AUTHOR
 Erik Fortune, Silicon Graphics
Index: xkbevd/xkbevd.man
===================================================================
--- xkbevd/xkbevd.man.orig	2007-02-08 17:12:52.000000000 +0100
+++ xkbevd/xkbevd.man	2007-02-08 17:14:08.000000000 +0100
@@ -3,98 +3,167 @@
 .\"
 .\" $XFree86: xc/programs/xkbevd/xkbevd.man,v 1.7 2001/01/27 18:21:16 dawes Exp $
 .\"
-.TH XKBEVD 1 __xorgversion__
+.TH xkbevd 1 __xorgversion__
 .SH NAME
 xkbevd \- XKB event daemon
 .SH SYNOPSIS
 .B xkbevd
-[ options ]
+.RI [ option \ ...]
 .SH DESCRIPTION
-.PP
-This command is very raw and is therefore only partially implemented;  we 
-present it here as a rough prototype for developers, not as a general purpose 
-tool for end users.  Something like this might make a suitable replacement
-for xev;  I'm not signing up, mind you, but it's an interesting idea.
+This command is very raw and is therefore only partially implemented;  we
+present it here as a rough prototype for developers, not as a general
+purpose tool for end users.
+Something like this might make a suitable replacement for
+.BR xev (1);
+I'm not signing up, mind you, but it's an interesting
+idea.
 .PP
 The
 .I xkbevd
-event daemon listens for specified XKB events and executes requested commands
-if they occur.   The configuration file consists of a list of event 
-specification/action pairs and/or variable definitions.   
-.PP
-An event specification consists of a short XKB event name followed by a 
-string or identifier which serves as a qualifier in parentheses;  empty 
-parenthesis indicate no qualification and serve to specify the default 
-command which is applied to events which do not match any of the other 
-specifications.  The interpretation of the qualifier depends on the type 
-of the event:  Bell events match using the name of the bell, message events 
-match on the contents of the message string and slow key events accept 
-any of \fIpress\fP, \fIrelease\fP, \fIaccept\fP, or \fIreject\fP.   No 
-other events are currently recognized.
-.PP
-An action consists of an optional keyword followed by an optional string 
-argument.  Currently, \fIxkbev\fP recognizes the actions: \fInone\fP, 
-\fIignore\fP, \fIecho\fP, \fIprintEvent\fP, \fIsound\fP, and \fIshell\fP.
+event daemon listens for specified XKB events and executes requested
+commands if they occur.
+The configuration file consists of a list of event specification/action
+pairs and/or variable definitions.
+.PP
+An event specification consists of a short XKB event name followed by a
+string or identifier which serves as a qualifier in parentheses; empty
+parentheses indicate no qualification and serve to specify the default
+command, which is applied to events which do not match any of the other
+specifications.
+The interpretation of the qualifier depends on the type of the event: bell
+events match using the name of the bell; message events match on the
+contents of the message string; and slow key events accept any of
+.BR press ,
+.BR release ,
+.BR accept ,
+or
+.BR reject .
+No other events are currently recognized.
+.PP
+An action consists of an optional keyword followed by an optional string
+argument.
+Currently,
+.B xkbevd
+recognizes the actions
+.BR none ,
+.BR ignore ,
+.BR echo ,
+.BR printEvent ,
+.BR sound ,
+and
+.BR shell .
 If the action is not specified, the string is taken as the name of a sound
 file to be played unless it begins with an exclamation point, in which case
 it is taken as a shell command.
 .PP
 Variable definitions in the argument string are expanded with fields from
 the event in question before the argument string is passed to the action
-processor.   The general syntax for a variable is 
-either $\fIc\P or $(\fIstr\fP), where \fIc\fP is a single character and
-\fIstr\fP is a string of arbitrary length.  All parameters have both 
-single-character and long names.    
-.PP
-The list of recognized parameters varies from event to event and is too long 
-to list here right now.   This is a developer release anyway, so you can
-be expected to look at the source code (evargs.c is of particular interest).
-.PP
-The \fIignore\fP, \fIecho\fP, \fIprintEvent\fP, \fIsound\fP,and \fIshell\fP
-actions do what you would expect commands named \fIignore\fP, \fIecho\fP,
-\fIprintEvent\fP, \fIsound\fP, and \fIshell\fP to do, except that the sound
-command has only been implemented and tested for SGI machines.   It launches
-an external program right now, so it should be pretty easy to adapt, 
-especially if you like audio cues that arrive about a half-second after you
-expect them.
+processor.
+The general syntax for a variable is either
+.RI $ c
+or
+.RI $( str ),
+where
+.I c
+is a single character and
+.I str
+is a string of arbitrary length.
+All parameters have both single\-character and long names.
+.PP
+The list of recognized parameters varies from event to event and is too long
+to list here right now.
+This is a developer release anyway, so you can be expected to look at the
+source code
+.RI ( evargs.c
+is of particular interest).
 .PP
-The only currently recognized variables are \fIsoundDirectory\fP and
-\fIsoundCmd\fP.  I'm sure you can figure out what they do.
+The
+.BR ignore ,
+.BR echo ,
+.BR printEvent ,
+.BR sound ,
+and
+.B shell
+actions do what you would expect commands named
+.BR ignore ,
+.BR echo ,
+.BR printEvent ,
+.BR sound ,
+and
+.B shell
+to do, except that the
+.B sound
+command has only been implemented and tested for SGI machines.
+It launches an external program right now, so it should be pretty easy to
+adapt, especially if you like audio cues that arrive about a half\-second
+after you expect them.
+.PP
+The only currently recognized variables are
+.I soundDirectory
+and
+.IR soundCmd .
+I'm sure you can figure out what they do.
 .SH OPTIONS
-.TP 8
-.B \-help
-Prints a usage message that is far more up-to-date than anything in this
-man page.
-.TP 8
-.B \-cfg \fIfile\fP
-Specifies the configuration file to read.   If no configuration file is
-specified, \fIxkbevd\fP looks for ~/.xkb/xkbevd.cf and $(LIBDIR)/xkb/xkbevd.cf
-in that order.
-.TP 8
-.B \-sc\ \fIcmd\fP
-Specifies the command used to play sounds.
-.TP 8
-.B \-sd\ \fIdirectory\fP
-Specifies a top-level directory for sound files.
-.TP 8
-.B \-display\ \fIdisplay\fP
-Specifies the display to use.  If not present, \fIxkbevd\fP uses $DISPLAY.
-.TP 8
+.TP
+.BR \-? ,\  \-help
+Print a usage message that is far more up\-to\-date than anything in this
+manpage (or not).
+.TP
 .B \-bg
-Tells \fIxkbevd\fP to fork itself (and run in the background).
-.TP 8
-.B \-synch
-Forces synchronization of all X requests.  Slow.
-.TP 8
+Fork and run in the background.
+.TP
+.BI \-cfg\  file
+Read the specified configuration file.
+If no configuration file is specified,
+.B xkbevd
+looks for
+.RI $HOME /.xkb/xkbevd.cf
+and
+.IR __projectroot__/lib/X11/xkb/xkbevd.cf ,
+in that order.
+.TP
+.BI \-d\  display\fR,\ \fP \-display\  display
+Connect to the X server at
+.IR display .
+If not specified,
+.B xkbevd
+uses
+.IR $DISPLAY .
+.TP
+.BR \-s ,\  \-synch
+Force synchronization of all X requests.
+Slow.
+.TP
+.BI \-sc\  cmd
+Use the command
+.I cmd
+to play sounds.
+.TP
+.BI \-sd\  directory
+Look for sound files at the file specification
+.I directory
+(should be an absolute path).
+.TP
 .B \-v
-Print more information, including debugging messages.   Multiple 
-specifications of \fI-v\fP cause more output, to a point.
+Print more information, including debugging messages.
+Multiple specifications of
+.B \-v
+cause more output, to a point.
 .SH "SEE ALSO"
-X(__miscmansuffix__)
+.BR X (7),
+.BR setxkbmap (1),
+.BR xkbbell (1),
+.BR xkbcomp (1),
+.BR xkbprint (1),
+.BR xkbvleds (1),
+.BR xkbwatch (1)
 .SH COPYRIGHT
-Copyright 1995, Silicon Graphics Computer Systems
-Copyright 1995, 1998  The Open Group
+Copyright 1995 Silicon Graphics Computer Systems
+.br
+Copyright 1995, 1998 The Open Group
 .br
-See \fIX(__miscmansuffix__)\fP for a full statement of rights and permissions.
+See
+.BR X (7)
+for a full statement of rights and permissions.
 .SH AUTHOR
 Erik Fortune, Silicon Graphics
Index: xkbprint/xkbprint.man
===================================================================
--- xkbprint/xkbprint.man.orig	2007-02-08 17:12:52.000000000 +0100
+++ xkbprint/xkbprint.man	2007-02-08 17:14:08.000000000 +0100
@@ -2,112 +2,198 @@
 .\"
 .\" $XFree86: xc/programs/xkbprint/xkbprint.man,v 1.7 2001/01/27 18:21:16 dawes Exp $
 .\"
-.TH XKBPRINT 1 __xorgversion__
+.TH xkbprint 1 __xorgversion__
 .SH NAME
 xkbprint \- print an XKB keyboard description
 .SH SYNOPSIS
-.B xkbprint 
-[options] source [ output_file ]
+.B xkbprint
+.RI [ option \ ...]
+.I source
+.RI [ output_file ]
 .SH DESCRIPTION
-.PP
 The
-.I xkbprint
-comman generates a printable or encapsulated PostScript description of the 
-XKB keyboard description specified by \fIsource\fP.  The \fIsource\fP can be 
-any compiled keymap (.xkm) file that includes a geometry description or an 
-X display specification.  If an \fIoutput_file\fP is specified, xkbprint 
-writes to it.  If no output file is specified, xkbprint creates replaces
-the extension of the source file with \fI.ps\fP or \fI.eps\fP depending
-on the requested format.  If the source is a non-local X display (e.g.:0), 
-xkbprint appends the appropriate prefix to the display specification, 
-replacing the colon with a dash.  For a local display, xkprint uses
-server-\fIn\fP where \fIn\fP is the number of the display.
+.B xkbprint
+command generates a printable or encapsulated PostScript description of the
+XKB keyboard description specifed by
+.IR source .
+.I source
+can be
+an X display specification or
+any compiled keymap
+.RI ( .xkm )
+file that includes a geometry description
+If
+.I output_file
+is specified,
+.B xkbprint
+writes to it.
+If no output file is specified, xkbprint creates a file of the name name as
+.IR source ,
+replacing the extension of the source file with
+.I .ps
+or
+.I .eps
+depending
+on the requested format.
+If
+.I source
+is a non\-local X display (e.g.,
+.BR :0 ),
+.B xkbprint
+appends the appropriate prefix to the display specification, replacing the
+colon with a dash.
+For a local display,
+.B xkbprint
+uses
+.RI server\- n ,
+where
+.I n
+is the server number of the display.
 .SH OPTIONS
-.TP 8
-.B \-?, -help
-Prints a usage message.
-.TP 8
+.TP
+.B \-?, \-help
+Print a usage message and exit.
+.TP
 .B \-color
-Print using the colors specified in the geometry file;  by default, xkbprint
-prints a black-and-white image of the keyboard.
-.TP 8
+Print using the colors specified in the geometry file; by default,
+.B xkbprint
+prints a black\-and\-white image of the keyboard.
+.TP
+.B \-d
+Report debugging information.
+This option is only available if debugging support was compiled into
+.BR xkbprint .
+.TP
 .B \-dflts
 Attempt to compute default names for any missing components, such as keys.
-.TP 8
+.TP
 .B \-diffs
 Show symbols only where they are explicitly bound.
-.TP 8
+.TP
 .B \-eps
 Generate an encapsulated PostScript file.
-.TP 8
+.TP
 .B \-fit
 Fit the keyboard image on the page (default).
-.TP 8
+.TP
 .B \-full
 Print the keyboard at full size.
-.TP 8
-.B \-grid\ \fIres\fP
-Print a grid with \fIres\fPmm resolution over the keyboard.
-.TP 8
-.B \-if\ \fIfontName\fP
-Specifies an internal PostScript type 1 font to dump to the specified
-output file or to \fIfontName\fP.pfa, if no output file is specified.
+.TP
+.BI \-grid\  res
+Print a grid over the keyboard with lines
+.IR res
+millimeters apart.
+.TP
+.BI \-I directory
+Use the specified directory (an absolute path) for include directives.
+This option may be specified multiple times, and is only available if
+debugging support was compiled into
+.BR xkbprint .
+.TP
+.BI \-if\  fontName
+Dump the specified font as an internal PostScript Type 1 font to the
+specified output file, or to
+.RI fontName .pfa
+if no output file is specified.
 No keyboard description is printed if an internal font is dumped.
-.TP 8
-.B \-label\ \fItype\fP
-Specifies the labels to be printed on keys;  legal types are: \fInone\fP,
-\fIname\fP,\fIcode\fP,\fIsymbols\fP.
-.TP 8
-.B \-lc\ \fI<locale>\fP
-Specifies a locale in which KeySyms should be resolved.
-.TP 8
+.TP
+.B \-kc
+Print keycodes, if possible.
+.TP
+.BI \-label\  type
+Print labels on the keys of the specified type;
+.BR none ,
+.BR name ,
+.BR code ,
+and
+.B symbols
+are accepted.
+.TP
+.BI \-lc\  locale
+Resolve keysyms in the specified locale.
+.TP
 .B \-level1
 Generate level 1 PostScript.
-.TP 8
+.TP
 .B \-level2
 Generate level 2 PostScript.
-.TP 8
-.B \-lg\ \fIgroup\fP
-Print symbols in keyboard groups starting from \fIgroup\fP.
-.TP 8
-.B \-ll\ \fIlevel\fP
-Print symbols starting from shift level \fIlevel\fP.
-.TP 8
+.TP
+.BI \-lg\  group
+Print symbols in keyboard groups starting from
+.IR group .
+.TP
+.BI \-ll\  level
+Print symbols starting from the specified shift level.
+.TP
 .B \-mono
-Generate black-and-white image of keyboard (default).
-.TP 8
-.B \-n\ \fInum\fP
-Print \fInum\fP copies.
-.TP 8
-.B \-nkg\ \fInum\fP
-Print the symbols in \fInum\fP keyboard groups.
-.TP 8
-.B \-npk\ \fInum\fP
-Number of keyboard images to print on each page; for EPS files, this 
-specifies the total number of keyboard images to print.
-.TP 8
-.B \-o\ \fIfile\fP
-Write output to \fIfile\fP.
-.TP 8
-.B \-R\fIdirectory\fP
-Use \fIdirectory\fP as the root directory; all path names are interpreted
-relative to \fIdirectory\fP.
-.TP 8
-.B -pict \fIwhich\fP
-Controls use of pictographs instead of keysym names where available. 
-\fIwhich\fP can be any of \fIall\fP, \fInone\fP, or \fIcommon\fP(default).
-.TP 8
-.B -synch
-Forces synchronization for X requests.
-.TP 8
-.B -w\ \fIlevel\fP
-Sets warning level (0 for no warning, 10 for all warnings).
+Generate black\-and\-white image of keyboard (default).
+.TP
+.BI \-n\  num
+Print
+.I num
+copies.
+.TP
+.BI \-nkg\  num
+Print the symbols in
+.I num
+keyboard groups.
+.TP
+.B \-nokc
+Do not print keycodes, even if it is possible.
+.TP
+.BI \-npk\  num
+Print
+.I num
+keyboard images on each page; for EPS files, this specifies the total
+number of keyboard images to print.
+.TP
+.BI \-ntg\  num
+Print no more than
+.I num
+total groups.
+.TP
+.BI \-o\  file
+Write output to
+.IR file .
+.TP
+.BI \-R directory
+Use
+.I directory
+as the root directory; all path names are interpreted relative to
+.IR directory .
+.TP
+.BI \-pict\  which
+Use pictographs (where available) instead of keysym names according to the
+specified policy.
+.I which
+can be any of
+.IR all ,
+.IR none ,
+or
+.I common
+(default).
+.TP
+.B \-synch
+Force synchronization of X requests.
+.TP
+.BI \-w\  level
+Set warning level to specified value (0 for no warning, 10 for all
+warnings).
 .SH "SEE ALSO"
-X(__miscmansuffix__),xkbcomp(1)
+.BR X (7),
+.BR setxkbmap (1),
+.BR xkbbell (1),
+.BR xkbcomp (1),
+.BR xkbevd (1),
+.BR xkbvleds (1),
+.BR xkbwatch (1)
 .SH COPYRIGHT
 Copyright 1995, Silicon Graphics Computer Systems
-Copyright 1995, 1998  The Open Group
 .br
-See \fIX(__miscmansuffix__)\fP for a full statement of rights and permissions.
+Copyright 1995, 1998 The Open Group
+.br
+See
+.BR X (7)
+for a full statement of rights and permissions.
 .SH AUTHOR
 Erik Fortune, Silicon Graphics
Index: setxkbmap/Makefile.am
===================================================================
--- setxkbmap/Makefile.am.orig	2007-02-08 17:12:52.000000000 +0100
+++ setxkbmap/Makefile.am	2007-02-08 17:14:47.000000000 +0100
@@ -58,7 +58,8 @@
 	-e 's|__libmansuffix__|$(LIB_MAN_SUFFIX)|g' \
 	-e 's|__adminmansuffix__|$(ADMIN_MAN_SUFFIX)|g' \
 	-e 's|__miscmansuffix__|$(MISC_MAN_SUFFIX)|g' \
-	-e 's|__filemansuffix__|$(FILE_MAN_SUFFIX)|g'
+	-e 's|__filemansuffix__|$(FILE_MAN_SUFFIX)|g' \
+	-e 's|__XKBCONFIGROOT__|$(XKBCONFIGROOT)|g'
 
 SUFFIXES = .$(APP_MAN_SUFFIX) .man
 
